cmd/tailscale/cli/up.go

```diff
@@ -44,6 +44,7 @@ specify any flags, options are reset to their default.
 		upf.BoolVar(&upArgs.acceptDNS, "accept-dns", true, "accept DNS configuration from the admin panel")
 		upf.BoolVar(&upArgs.singleRoutes, "host-routes", true, "install host routes to other Tailscale nodes")
 		upf.StringVar(&upArgs.exitNodeIP, "exit-node", "", "Tailscale IP of the exit node for internet traffic")
+		upf.BoolVar(&upArgs.allowLocalAccess, "allow-local-access", false, "Allow direct access to the local network when using exit nodes")
```

Comment:
neither the flag name or help text make it clear which direction this is for (as a client of an exit node? or being an exit node myself?)

also, later we'll want the other one too.

Reply:
Updated the help text

Reply:
I think I want something that starts with `exit-node` in the flag name so it sorts next to it and it's super obvious from reading the flag.

Like `exit-node-blocks-local-access`, defaulting to `true`, but let users `tailscale set exit-node-blocks-local-access=false`. (or for now: `tailscale up --exit-node-blocks-local-access=false`)

Thoughts, @danderson?

Reply:
(then when we add the other option, it can be named something like `--advertise-exit-node-allows-other-advertised-ranges` or whatever)

Reply:
exit-node-blocks-local-access seems okay. We could also reverse it, `exit-node-allow-lan-access`. Depends what we think the default should be, IMO that should be the =false value.

Reply:
is allow/block really the user intention? or should we be more explicit like --exit-node-route-local-network?

Reply:
or should it be more generic like `--route-local-network` so that it works with subnet routes as well.

Reply:
Went with `exit-node-allow-lan-access`